source/js/components/join/join.jsx

```diff
@@ -468,8 +468,7 @@ export default class JoinUs extends React.Component {
    * Render the submit button in signup CTA.
    */
   renderSubmitButton() {
-    let classnames = classNames("btn btn-primary", {
-      "w-100": !this.isFlowForm(),
+    let classnames = classNames("btn btn-secondary", {
```

Comment:
Please revert this. Designers are fine with keepign the button as btn-primary. 👍